src/app/api/import-to-sanity/route.ts

```
@@ -45,55 +46,87 @@
+      name: transformed.name,
+      slug: transformed.slug,
+      heading: transformed.heading,
+      subheading: transformed.subheading
+    }
+    return pageDoc
+  }
+}
+
+// Helper function to process media references in content
+function processMediaInContent(
+  content: any[],
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)